docker-compose.yml

```diff
@@ -62,12 +62,27 @@ services:
       context: ./src/backend/oracle-data-interface
       args:
         - SKIP_TESTS=true
-        - ENABLE_SPLUNK=${ENABLE_SPLUNK:-false}
+        - SPLUNK_URL=${SPLUNK_URL:-https://splunk:8088}
+        - SPLUNK_TOKEN=${SPLUNK_HEC_TOKEN:-token}
     ports:
       - "5010:8080"
-    networks:
-      - traffic-court-net
 
+  #############################################################################################
+  ###                           Oracle Interface API                                        ###
```

Comment:
I think the title should be "Splunk" for this section.

Reply:
Quite right.  Good catch. Fixed.